x-pack/plugins/index_management/public/application/services/documentation.ts

```diff
@@ -10,45 +10,126 @@ import { DataType } from '../components/mappings_editor/types';
 import { TYPE_DEFINITION } from '../components/mappings_editor/constants';
 
 class DocumentationService {
+  private dataStreams: string = '';
   private esDocsBase: string = '';
-  private kibanaDocsBase: string = '';
-
+  private indexManagement: string = '';
+  private indexSettings: string = '';
+  private indexTemplates: string = '';
+  private indexV1: string = '';
+  private mapping: string = '';
+  private mappingAnalyzer: string = '';
+  private mappingCoerce: string = '';
+  private mappingCopyTo: string = '';
+  private mappingDocValues: string = '';
+  private mappingDynamic: string = '';
+  private mappingDynamicFields: string = '';
+  private mappingDynamicTemplates: string = '';
+  private mappingEagerGlobalOrdinals: string = '';
+  private mappingEnabled: string = '';
+  private mappingFieldData: string = '';
+  private mappingFieldDataFilter: string = '';
+  private mappingFieldDataTypes: string = '';
+  private mappingFieldDataEnable: string = '';
+  private mappingFormat: string = '';
+  private mappingIgnoreAbove: string = '';
+  private mappingIgnoreMalformed: string = '';
+  private mappingIndex: string = '';
+  private mappingIndexOptions: string = '';
+  private mappingIndexPhrases: string = '';
+  private mappingIndexPrefixes: string = '';
+  private mappingJoinFieldsPerformance: string = '';
+  private mappingMeta: string = '';
+  private mappingMetaFields: string = '';
+  private mappingNormalizer: string = '';
+  private mappingNorms: string = '';
+  private mappingNullValue: string = '';
+  private mappingParameters: string = '';
+  private mappingPositionIncrementGap: string = '';
+  private mappingRankFeatureFields: string = '';
+  private mappingRouting: string = '';
+  private mappingSimilarity: string = '';
+  private mappingSourceFields: string = '';
+  private mappingSourceFieldsDisable: string = '';
+  private mappingStore: string = '';
+  private mappingTermVector: string = '';
+  private mappingTypesRemoval: string = '';
+  private percolate: string = '';
+  private runtimeFields: string = '';
   public setup(docLinks: DocLinksStart): void {
-    const { DOC_LINK_VERSION, ELASTIC_WEBSITE_URL } = docLinks;
-    const docsBase = `${ELASTIC_WEBSITE_URL}guide/en`;
-
-    this.esDocsBase = `${docsBase}/elasticsearch/reference/${DOC_LINK_VERSION}`;
-    this.kibanaDocsBase = `${docsBase}/kibana/${DOC_LINK_VERSION}`;
+    const { links } = docLinks;
+    this.dataStreams = `${links.elasticsearch.dataStreams}`;
```

Comment:
[nit] it's not necessary to use template strings here (and code below)

```suggestion
    this.dataStreams = links.elasticsearch.dataStreams;
```